### PR TITLE
update README to cover clean up.

### PR DESCRIPTION
**Pull request by busbey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -161,6 +166,13 @@ to administer Fluo:
 If you run some tests and then want a fresh cluster, run the `setup` command again which will
 kill all running processes, clear any data and logs, and restart your cluster.
 
+## Wipe command
+
+The `wipe` command will kill all running processes for your local development cluster and clear
```
Comment (cjmctague) [inline README.md]: `uno wipe`

File: README.md [hunk 2]
```diff
@@ -161,6 +166,13 @@ to administer Fluo:
...
+all the data and logs. It does *not* delete the binary tarballs downloaded by the `fetch` command
```
Comment (cjmctague) [inline README.md]: Could mention that the data and logs are cleared from the Install directory.

File: README.md [hunk 3]
```diff
@@ -161,6 +166,13 @@ to administer Fluo:
...
+so you can use `setup` directly again in the future. If you need to reclaim the space used by
+the binary tarballs you'll have to manually delete them.
```
Comment (cjmctague) [inline README.md]: Could mention that the tarballs are located in the download folder.
`~/path/to/uno/downloads`

**Comment by busbey** [comment]
I wasn't sure if the specifics of the `install` and `downloads` directories would be too into internal implementation details to include in the top level README. if you think that's fine I can add it.

**Comment by cjmctague** [comment]
Now that I think about it again, probably best to leave them out. They are configurable and wouldn't match everyone's setup if the defaults were changed.

File: README.md (hunk 1 above)
Comment (busbey) [reply README.md]: done and updated the setup command to match.

File: README.md (hunk 2 above)
Comment (busbey) [reply README.md]: internal details should be left out, per conversation in review

File: README.md (hunk 3 above)
Comment (busbey) [reply README.md]: internal details should be left out per conversation in review.

**Comment by keith-turner** [comment]
@busbey thanks for the contribution.  Would you like to be added as a [contributor](http://fluo.apache.org/people/)?  If so, submit a [PR](https://github.com/apache/fluo-website/tree/gh-pages) or let me know what info to add.

Also, new contributors are mentioned by [@ApacheFluo](https://twitter.com/apachefluo) if they like.   If you are interested let me know, will only tweet if its ok with you.

**Pull request merged**